src/Row.js

```diff
@@ -156,7 +156,7 @@ export default class Row extends Component {
     return (
       <Animated.View
         {...this._panResponder.panHandlers}
-        style={[style, styles.container, this._animatedLocation.getLayout()]}
+        style={[style, styles.container, {transform: this._animatedLocation.getTranslateTransform()}]}
```

Comment:
I didn't notice any change in animation performance. why it has to improve it, is it mentioned anywhere? I know that in browsers transforms give much smoother animation than positioning with top/left, but as for as I know it is not true for react native